recipes/tiddit/build.sh

```diff
@@ -2,12 +2,15 @@
 outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/lib
 echo $PREFIX
 
 mkdir build
 cd build
 cmake ..
 make
+cp ../bin/TIDDIT $PREFIX/bin
```

Comment:
Should a `tiddit` symlink for the be created?

Reply:
I don't think so, they don't seem to use it in their documentation that I noticed. I think I added that last time by accident.

Reply:
Okay, then change the test to use `TIDDIT`?

Reply:
Crap, I forgot to reenable the tests. Nevermind on this one.

Reply:
Actually that won't hurt anything and I guess they mention it in the conda section.

Reply:
Yes, no, what? Now I'm confused, hehe.

Well, in their docs they mention `tiddit` in the context of Bioconda: https://github.com/SciLifeLab/TIDDIT/blame/TIDDIT-2.8.1/README.md#L36

So, probably run `tiddit --help` in the tests to make sure that works?

Reply:
It's a symlink to the python script :)

Reply:
Yeah, I saw that. I just meant testing `tiddit` to avoid regressions if that symlink might be accidentally removed in the future.